repl/index.js

```diff
@@ -1,24 +1,23 @@
-var vm = require("vm");
-var compile = require("../node/caching-compiler.js").compile;
+const vm = require("vm");
+const compile = require("../node/caching-compiler.js").compile;
 
 require("../node");
 
 function wrap(name, optionsArgIndex) {
-  var method = vm[name];
+  const method = vm[name];
 
-  if (typeof method !== "function") {
-    return;
-  }
-
-  if (method.reified) {
+  if (typeof method !== "function" ||
+      method.reified) {
     return;
   }
 
   vm[name] = function (code) {
```

Comment:
Ah, it's because arrow functions don't have `arguments` objects so it's referencing the `arguments` object of the module itself.